### PR TITLE
make konfigurator configurable to watch all namespaces

### PR DESCRIPTION
**Pull request by kahootali**

### CODE REVIEW

File: cmd/Konfigurator/main.go [hunk 1]
```diff
@@ -36,14 +36,22 @@ func main() {
 }
 
 func watchKonfiguratorTemplate() {
-	namespace, err := k8sutil.GetWatchNamespace()
+	namespace, err := getWatchNamespace()
```
Comment (waseem-h) [inline cmd/Konfigurator/main.go]: This will give a syntax error

File: cmd/Konfigurator/main.go [hunk 2]
```diff
@@ -36,14 +36,19 @@ func main() {
...
-	namespace, err := k8sutil.GetWatchNamespace()
-	if err != nil {
-		logrus.Fatalf("Failed to get watch namespace: %v", err)
-	}
+	namespace := getWatchNamespace()
 
 	watch("konfigurator.stakater.com/v1alpha1", "KonfiguratorTemplate", namespace, 15)
 }
 
+func getWatchNamespace() string {
```
Comment (waseem-h) [inline cmd/Konfigurator/main.go]: I think its better to reuse "GetWatchNamespace" method from k8sutil package. We can just make the error a warning, saying that no WATCH_NAMESPACE detected so watching in all namespaces. And the rest will work as expected

**Comment by stakater-user** [comment]
@kahootali Image is available for testing. ``docker pull stakater/konfigurator:PR-23-4``

File: cmd/Konfigurator/main.go (hunk 2 above)
Comment (kahootali) [reply cmd/Konfigurator/main.go]: https://github.com/operator-framework/operator-sdk/blob/121664e0093774c6d4fa4528fe7367d0848e6d74/pkg/k8sutil/k8sutil.go#L36

They log an error in their function as well which is not good as if our app logs an error but we are supporting that as a feature.

**Comment by stakater-user** [comment]
@kahootali Image is available for testing. ``docker pull stakater/konfigurator:PR-23-5``

**Comment by waseem-h** [comment]
looks good @kahootali

**Comment by stakater-user** [comment]
@kahootali Image is available for testing. ``docker pull stakater/konfigurator:PR-23-6``

**Pull request merged**